index.bs

```diff
@@ -720,10 +720,31 @@ An {{XRSpace}} describes an entity that is tracked by the [=/XR device=]'s track
+
+The [=effective origin=] of an {{XRSpace}} can only be observed in the coordinate system of another {{XRSpace}} as an {{XRPose}}, returned by an {{XRFrame}}'s {{XRFrame/getPose()}} method. The spatial relationship between {{XRSpace}}s MAY change between {{XRFrame}}s.
+
+<div class="algorithm" data-algorithm="populate-the-pose">
+
+To <dfn>populate the pose</dfn> of a {{XRSpace}} |sourceSpace| in an  {{XRSpace}} |destinationSpace| at the time represented by a {{XRFrame}} |frame| into an {{XRPose}} |pose|, the user agent MUST run the following steps:
+
+  1. 1. If |frame|'s [=active=] boolean is <code>false</code>, throw an {{InvalidStateError}} and abort these steps.
+  1. Let |session| be |frame|'s {{XRFrame/session}} object.
+  1. If |sourceSpace|'s [=XRSpace/session=] does not equal |session|, throw an {{InvalidStateError}} and abort these steps.
+  1. If |destinationSpace|'s [=XRSpace/session=] does not equal |session|, throw an {{InvalidStateError}} and abort these steps.
+  1. If |destinationSpace|'s pose cannot be determined relative to |sourceSpace| at the time represented by |frame|, set |pose| to <code>null</code>.
+  1. Let |transform| be |pose|'s {{XRPose/transform}}.
+  1. Set |transform|'s {{XRRigidTransform/position}} to the location of |sourceSpace|'s [=effective origin=] in |destinationSpace|'s [=coordinate system=].
+  1. Set |transform|'s {{XRRigidTransform/orientation}} to the orientation of |sourceSpace|'s [=effective origin=] in |destinationSpace|'s [=coordinate system=].
+
+</div>
+
 XRReferenceSpace {#xrreferencespace-interface}
 ------------------
 
-An {{XRReferenceSpace}} describes an {{XRSpace}} that is generally expected to remain static for the duration of the {{XRSession}}, with the most common exception being mid-session reconfiguration by the user. Every {{XRReferenceSpace}} describes a coordinate system where the Y axis MUST be aligned with gravity, with <code>+Y</code> being "Up". <code>-Z</code> is considered "Forward", and <code>+X</code> is considered "Right".
+An {{XRReferenceSpace}} is one of several common {{XRSpace}}s that applications can use to establish a spatial relationship with the user's physical environment.
+
+{{XRReferenceSpace}}s are generally expected to remain static for the duration of the {{XRSession}}, with the most common exception being mid-session reconfiguration by the user. The [=native origin=] for every {{XRReferenceSpace}} describes a coordinate system where <code>+X</code> is considered "Right", <code>+Y</code> is considered "Up", and <code>-Z</code> is considered "Forward". All {{XRReferenceSpace}}s except {{XRReferenceSpaceType/identity}} MUST align the Y axis with gravity.
```

Comment:
Why "must align Y with gravity"?  This seems unnecessary.  I would expect that perhaps we want to align with the Y axis of the underlying system.  But, consider doing webxr in a vehicle:  generally, the tracking system aligns with the vehicle, not the world (this is especially true on fast moving systems like airplanes).

Worse, what about space?  NASA has sent Hololens' devices to the ISS;  I am working with the company that builds the maintenance system for the ISS, which is web-based, to add AR instructions via WebXR.  We cannot require that "Y" is aligned with gravity.

Reply:
worth noting: the native origin is never directly exposed, so this doesn't actually matter much aside from the device's tracking system having _some_ notion of down that is consistent between spaces on a per-frame basis

but yeah, i think we can get away with not talking about gravity at all and just using the deliberately underdefined terms right/up/forward